save-orchestrator-common/src/main/kotlin/com/saveourtool/save/orchestrator/kubernetes/KubernetesManager.kt

```diff
@@ -98,7 +100,11 @@ class KubernetesManager(
     override fun cleanupAllByExecution(executionId: Long) {
         logger.debug { "Removing a job for execution id=$executionId" }
         val jobName = jobNameForExecution(executionId)
-        val job = kcJobsWithName(jobName)
+        val job = kc.batch()
+            .v1()
+            .jobs()
+            .inNamespace(kubernetesSettings.agentNamespace)
```

Comment:
Does it change anything? As far as I remember, I already have declared a namespace in Job metadata

Reply:
Oh sure, sorry, my bad